pydsa/__init__.py

```diff
@@ -17,4 +17,4 @@
 from .gnome_sort import gnome_sort
 from .mod_inverse import mod_inverse
 from .euclid import gcd, extended_euclid
-from .counting_inversions import counting_inversions
+from .counting_inversions import counting_inversions
```

Comment:
Hey I already updated that last night and verified with online pep8 checker.
Kindly check the updated code. It has a newline at the end of file.